draft-ietf-httpbis-client-hints.md

```diff
@@ -190,6 +200,12 @@ Implementers ought to consider both user and server controlled mechanisms and po
 Implementers SHOULD support Client Hints opt-in mechanisms and MUST clear persisted opt-in preferences when any one of site data, browsing history, browsing cache, or similar, are cleared.
 
 
+## Abuse Detection
+A user agent that tracks access to active fingerprinting information SHOULD consider emission of Client Hints headers similarly to the way it would consider access to the equivalent API.
+
+User agents which detect abuse through forensic means can observe HTTP responses to requests containing Client Hints, and their variance based on those hints. As Client Hints are provided in order to allow variation and optimization of the content to the provided Client-Hints, resource responses that do not vary on the Client Hints provided can indicate that such Hints are being used for nefarious purposes.
```

Comment:
> resource responses that do not vary on the Client Hints provided can indicate that such Hints are being used for nefarious purposes

Hmm.. I can see `no-cache` + `no-store` responses not emitting Vary by default today, and that doesn't directly imply "nefarious" use. If we're making that claim, perhaps we should also be recommending that servers always emit such Vary responses, regardless of cacheability of the response? Or, am I overthinking it?

Reply:
I'm not talking about using the `Vary` header, but about actually varying the resource served when different hints are included in the request. Maybe I should clarify that?

Reply:
Ah, I see. Yeah, I was picturing an implementation looking for `Vary` in the response to asses use: seems like a straightforward + explicit signal? Probably worth being (slightly) more explicit.

Reply:
Added another sentence to clarify that.